web-app/src/common/prediction.spec.ts

```diff
@@ -0,0 +1,116 @@
+import expect from 'expect';
+import { Coordinates, parseGameStateFromMultilineString } from './utils';
+import { getNextPlaySuggestion } from './prediction';
+
+function areCoordinatesEquals(coord1: Coordinates, coord2: Coordinates) {
+  return coord1.x === coord2.x && coord1.y === coord2.y;
+}
+
+describe('Should get an advice for the next play', () => {
+  it('Should get a good advice', () => {
+    const input = parseGameStateFromMultilineString(`
+  ⬡ ⬡ ⬡
+   ⬡ ⬡ ⬡
+    ⬢ ⬢ ⬡
+  `);
+
+    const nextPlayAdvice = getNextPlaySuggestion(input, "black");
+
+    expect(areCoordinatesEquals(nextPlayAdvice, { x: 2, y: 2 }) || areCoordinatesEquals(nextPlayAdvice, { x: 2, y: 1 })).toBeTruthy();
+  });
+
+  it('Should get a good advice', () => {
+    const input = parseGameStateFromMultilineString(`
+  ⬡ ⬡ ⬡
+   ⬡ ⬡ ⬡
+    ⬢ ⬡ ⬢
+  `);
+
+    const nextPlayAdvice = getNextPlaySuggestion(input, "black");
+
+    expect(areCoordinatesEquals(nextPlayAdvice, { x: 1, y: 2 }) || areCoordinatesEquals(nextPlayAdvice, { x: 1, y: 1 })).toBeTruthy();
+  });
+
+
+  it('Should get a good advice', () => {
+    const input = parseGameStateFromMultilineString(`
+  ⬡ ⬡ ⬡
+   ⬡ ⬡ ⬡
+    ⬡ ⬢ ⬢
+  `);
+
+    const nextPlayAdvice = getNextPlaySuggestion(input, "black");
+
+    expect(areCoordinatesEquals(nextPlayAdvice, { x: 0, y: 2 }) || areCoordinatesEquals(nextPlayAdvice, { x: 0, y: 1 })).toBeTruthy();
+  });
+
+  it('Should get a good advice', () => {
+    const input = parseGameStateFromMultilineString(`
+  ⬡ ⬡ ⬡
+   W ⬡ ⬡
+    W ⬡ ⬡
+  `);
+
+    const nextPlayAdvice = getNextPlaySuggestion(input, "white");
+
+    expect(areCoordinatesEquals(nextPlayAdvice, { x: 0, y: 0 }) || areCoordinatesEquals(nextPlayAdvice, { x: 1, y: 0 })).toBeTruthy();
+  });
+
+  it('Should get a good advice', () => {
+    const input = parseGameStateFromMultilineString(`
+  W ⬡ ⬡
+   ⬡ ⬡ ⬡
+    W ⬡ ⬡
+  `);
+
+    const nextPlayAdvice = getNextPlaySuggestion(input, "white");
+
+    expect(areCoordinatesEquals(nextPlayAdvice, { x: 0, y: 1 }) || areCoordinatesEquals(nextPlayAdvice, { x: 1, y: 1 })).toBeTruthy();
+  });
+
+  it('Should get a good advice', () => {
+    const input = parseGameStateFromMultilineString(`
+  W ⬡ ⬡
+   W ⬡ ⬡
+    ⬡ ⬡ ⬡
+  `);
+
+    const nextPlayAdvice = getNextPlaySuggestion(input, "white");
+
+    expect(areCoordinatesEquals(nextPlayAdvice, { x: 0, y: 2 }) || areCoordinatesEquals(nextPlayAdvice, { x: 1, y: 2 })).toBeTruthy();
+  });
+
+  it('Should get a good advice', () => {
+    const input = parseGameStateFromMultilineString(`
+  W ⬡ ⬡
+   ⬡ W ⬡
+    ⬡ ⬡ ⬡
+  `);
+
+    const nextPlayAdvice = getNextPlaySuggestion(input, "white");
+
+    expect(areCoordinatesEquals(nextPlayAdvice, { x: 1, y: 2 }) || areCoordinatesEquals(nextPlayAdvice, { x: 2, y: 2 })).toBeTruthy();

```

Comment:
Here it's different, I would expect `{ x: 0, y: 2 }` or `{ x: 1, y: 2 }` but not `{ x: 2, y: 2 }`

Reply:
I think `{ x: 1 y: 0 }` , `{ x: 2 y: 0}` and `{ x: 0 y: 1}` are valid too !